src/Microsoft.Identity.Web.TokenCache/Microsoft.Identity.Web.TokenCache.csproj

```diff
@@ -6,6 +6,7 @@
     <Description>This package bring token cache serializers for MSAL.NET confidential client applications.</Description>
     <ProjectGuid>{7885DFBB-0D20-4115-86E2-709C2E12253B}</ProjectGuid>
     <PackageReadmeFile>README.md</PackageReadmeFile>
+    <IsAotCompatible>false</IsAotCompatible>
```

Comment:
I would like to understand better the error here.

Reply:
We want to avoid having to set this to false in as many areas as possible.